packages/serialization/json/src/jsonSerializationWriter.ts

```diff
@@ -43,36 +43,16 @@ export class JsonSerializationWriter implements SerializationWriter {
       isValuePresent &&
       this.writer.push(JsonSerializationWriter.propertySeparator);
   };
-  public writeNumberValue = (key?: string, value?: number): void => {
-    key && value && this.writePropertyName(key);
-    value && this.writer.push(`${value}`);
-    key && value && this.writer.push(JsonSerializationWriter.propertySeparator);
-  };
+  public writeNumberValue = (key?: string, value?: number): void => this.writeStringValue(key, value?.toString());
```

Comment:
sorry I missed that one, numbers don't need quotes, can you revert this one too please?

Reply:
Done.